src/JHipsterNetSampleApplication/Controllers/LabelController.cs

```diff
@@ -64,7 +64,8 @@ public async Task<IActionResult> UpdateLabel([FromBody] Label label)
         public ActionResult<IEnumerable<Label>> GetAllLabels(IPageable pageable)
         {
             _log.LogDebug("REST request to get a page of Labels");
-            var page = _applicationDatabaseContext.Labels.UsePageable(pageable);
+            var page = _applicationDatabaseContext.Labels
+                .UsePageable(pageable);
```

Comment:
You did the newline on-purpose or it's the IDE formatting?

Reply:
I think it's me because I saw everywhere the queries on this format : 
await _applicationDatabaseContext.Operations
                .Include(operation => operation.BankAccount)
                    .ThenInclude(bankAccount => bankAccount.User)
                .Include("OperationLabels.Label")
I thought it was more readable so I made these changes. I can find them and undo it if you prefer ?

Reply:
You're right the newline is better. I anticipate the code generation and the 
```
dbContext.{{Entity}}
//Includes
.Include({{OtherEntity}})
.ThenInclude({{AnotherOne}})
//Termination
{{UsePageable}}
```